docs/source/guide/explanation/algorithms/object_detection/object_detection.rst

```diff
@@ -17,7 +17,49 @@ To be added soon
 Semi-supervised Learning
 ************************
 
-To be added soon
+Semi-SL (Semi supervised Learning) is a type of machine learning algorithm that uses both labeled and unlabeled data to improve the performance of the model. This is particularly useful when labeled data is limited, expensive or time-consuming to obtain.
+
+`Unbiased Teacher model <https://arxiv.org/abs/2102.09480>`_ is a specific implementation of Semi-SL for object detection. Unbiased teacher detach student model and teacher model to prevent teacher from being polluted by noisy pseudo labels. In the early stage, teacher model trained by supervised loss. This stage is called burn-in stage. After burn-in the student model is trained using both psudo labeled data from teacher model and labeled data. And the teacher model is updated using
```

Comment:
```suggestion
For Semi-SL task solving we use `Unbiased Teacher model <https://arxiv.org/abs/2102.09480>`_ is a specific implementation of Semi-SL for object detection. Unbiased teacher detach the student model and teacher model to prevent teacher from being polluted by noisy pseudo-labels. In the early stage, the teacher model is trained by supervised loss. This stage is called a burn-in stage. After the burn-in, the student model is trained using both pseudo-labeled data from the teacher model and labeled data. And the teacher model is updated using
```